isis/src/control/objs/BundleResults/BundleResults.cpp

```diff
@@ -732,21 +817,40 @@ namespace Isis {
   }
 
 
+  /**
+   * Sets the total number of lidar range constraints.
+   *
+   * @param numberLidarRangeConstraints The total number of lidar range constraints.
+   */
+  void BundleResults::setNumberLidarRangeConstraints(int numberLidarRangeConstraints) {
+    m_numberLidarRangeConstraintEquations = numberLidarRangeConstraints;
+  }
+
+
   /**
    * Computes the degrees of freedom of the bundle adjustment and stores it internally.
    */
   void BundleResults::computeDegreesOfFreedom() {
-    m_degreesOfFreedom = m_numberObservations
+    m_degreesOfFreedom = m_numberObservations //TODO remove m_numberObservations
```

Comment:
I think having this in here is technically incorrect since m_numberobservations is calculated by adding together m_numberImageObservations and m_number LidarImageObservations

Reply:
line 1321